x-pack/plugins/apm/server/routes/alerts/register_error_count_alert_type.test.ts

```diff
@@ -144,6 +144,8 @@ describe('Error count alert', () => {
       triggerValue: 5,
       reason: 'Error count is 5 in the last 5 mins for foo. Alert when > 2.',
       interval: '5m',
+      viewInAppUrl:
+        'http://localhost:5601/eyr/app/apm/services/foo/errors?environment=env-foo',
```

Comment:
Correct me if I'm wrong but the `../eyr/...` is different depending on the cluster where we run the test, isn't it?

Reply:
I tested on my local, all good 👍